app/components/validated-input/component.js

```diff
@@ -0,0 +1,43 @@
+import Ember from 'ember';
```

Comment:
For now I've just included a link to the ember-cp-validations github repo to the top of the file.